app/src/main/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModel.kt

```diff
@@ -35,7 +36,17 @@ class ProfileViewModel @Inject constructor(
 		}
 	}
 
+	fun updateAvatar(uri: Uri) {
+		viewModelScope.launch {
+			_profile.value = Loading
+			authRepository.updateUserAvatar(uri)
+			fetchProfile()
```

Comment:
same situation like in Your bench project :D

Reply:
Yes, the same situation mabey I can find a solution here 😄